src/main/java/io/bittiger/ads/web/SearchResult.java

```diff
@@ -123,11 +125,17 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                     out.write("Keywords: "+ key.toString()+"\n");
                     String bid = String.valueOf(main.getLong(BID));
                     out.write("Bid: "+ bid +"\n");
-                    String relevantScore = String.valueOf(main.getDouble("relevantScore"));
+                    double value = main.getDouble("relevantScore");
```

Comment:
You can put "relevantScore" to config file and make it as a constant. Same as qualityScore below. 😉